machines/store.ts

```diff
@@ -405,9 +409,12 @@ export function encryptJson(encryptionKey: string, data: string): string {
   return CryptoJS.AES.encrypt(data, encryptionKey).toString();
 }
 
-export function decryptJson(encryptionKey: string, encrypted: string): string {
+export function decryptJson(
```

Comment:
can we avoid exporting decryptJson if it is only required within store.ts but not outside it?